src/FileInput/FileInput.jsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import Dropzone from "react-dropzone";
+import {FlexBox, FlexItem} from "../flex";
+
+require("./FileInput.less");
+
+function DefaultIcon() {
+  return (<svg className="FileInput--Icon" width="42px" height="24px" viewBox="0 0 42 24">
+    <g>
+      <g transform="translate(-242.000000, -13.000000)" fill="#566279">
```

Comment:
We still don't have a great story on how to handle SVG styling and content so it is fine to just not worry about it for now